### DISCUSSION TITLE
File placement inside container

### DISCUSSION THREAD

**Discussion by dnhodgson** [concourse/concourse#6160] [Help & Support] [answered]
I'm trying to using concourse to do some configuration testing of an application that's already packed into a container. The application expects the config files to be in a few different locations inside the container, is it possible to map specific files from the git resource to locations inside the container?

Example would be something like
```
git_resouce/configs/configA => /etc/myapp/configA
git_resouce/configs/configB => /usr/local/myapp/configB
```

**Comment by jamieklassen** [comment, marked as answer]
There are probably a lot of ways to achieve something along these lines, but it's not a workflow with an explicit feature supporting it. Are you asking for general advice before starting, or can you give an example of what you've tried so far in an existing pipeline/job?

**Reply by dnhodgson** [reply]
I ended up creating a script that gets launched as the entrypoint in the container. The script stages the config files in their correct locations. This seems to work well.
